troubleshooting.rst

```diff
@@ -0,0 +1,8 @@
+Troubleshooting
+***************
+
+Install issues getting you down?
+Error messages making you frown?
+If you've been trying for awhile and can't figure out,
+`shoot me an email <mailto:trouble@nstack.com>`_
```

Comment:
I don't know what you think of this, but I thought it could be a good concierge troubleshooting